src/components/RemediationWizardV2/ModalStatusContent.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Button,
+  Flex,
+  Spinner,
+  Bullseye,
+} from '@patternfly/react-core';
+
+const ModalStatusContent = ({
```

Comment:
This is mostly freeform content rendering in preperation for when the UX work for the copy is actually complete